docs/docs/getting-started/managing-confidence.md

```diff
@@ -6,7 +6,7 @@ The desired confidence level is set as the escalation threshold on your detector
 
 For example, say you want to set your desired confidence level to 0.95, but that you're willing to wait up to 60 seconds to get a confident response.
 
-<!-- TODO: turn on `confidence` parameter, and re-enable tests! -->
+<!-- We skip tests here because the tests may be slow -->
```

Comment:
I did run this doc test, but I'm not sure we want to run it in our test matrix (which would run this 20x).

Reply:
We should think about enabling this on at least some of the platforms.  Like maybe just 3.10.

Reply:
Actually it would make sense to only run some of the tests when PIL is installed, per earlier discussion.

Reply:
I made some improvements in https://github.com/groundlight/python-sdk/pull/40